modules/javafx.web/src/main/java/com/sun/javafx/webkit/prism/WCGraphicsPrismContext.java

```diff
@@ -442,6 +442,11 @@ public void setClip(WCRectangle c) {
                               (int)c.getWidth(), (int)c.getHeight()));
     }
 
+    public void setClip(int cx, int cy, int cw, int ch, WCImage maskImage) {
+        setClip(new Rectangle(cx, cy, cw, ch));
+        state.setClipMaskImage(maskImage);
```

Comment:
Should all of the other variants of setClip (the ones that don't take a maskImage) set the clipMaskImage to null? Otherwise it seems that it might not be reset to null in all cases.

Reply:
added setting of null to maskImage for all the overloads of setClip where mask image is not present.